src/generators.ts

```diff
@@ -83,7 +83,16 @@ export class FolderGenerator<T> implements IGenerator<T> {
     innterContext.workingDirectory = path.join(context.workingDirectory, folderName);
     innterContext = resolveContextModifier(innterContext, this._contextMap);
 
-    await fs.mkdir(innterContext.workingDirectory);
+    if((context.options as any).overWriteDir === true) {
+      await fs.rmdir(innterContext.workingDirectory);

```

Comment:
please, check exceptions, it will happens for example case when user hasn't permissions for delete